modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -429,13 +429,28 @@ else if (value instanceof Object[]) {
 
 						sb.append(stringArray);
 					}
-					else {
-						if (type.equals(DDMFormFieldTypeConstants.RICH_TEXT)) {
-							valueString = _htmlParser.extractText(valueString);
-						}
+					else if (type.equals(DDMFormFieldTypeConstants.RICH_TEXT)) {
```

Comment:
@rolandpakai could you please sort the `if` statements alphabetically?

```
if (type.equals(
    DDMFormFieldTypeConstants.DOCUMENT_LIBRARY) ||
    type.equals(DDMFormFieldTypeConstants.IMAGE) ||
    type.equals(
        DDMFormFieldTypeConstants.JOURNAL_ARTICLE) ||
    type.equals(
        DDMFormFieldTypeConstants.LINK_TO_LAYOUT)) {

} else if (type.equals(DDMFormFieldTypeConstants.RICH_TEXT)) {

} else if (type.equals(DDMFormFieldTypeConstants.SELECT)) {

} else {
    
}
```
Thanks!

Reply:
Yes. sure